diff --git a/src/audio_engine/CMakeLists.txt b/src/audio_engine/CMakeLists.txt
index b78bff2..d8e3919 100644
--- a/src/audio_engine/CMakeLists.txt
+++ b/src/audio_engine/CMakeLists.txt
@@ -75,7 +75,7 @@ macro(add_ae_test TEST_NAME TEST_SRC LIB_TARGET)
         )
     endif()
 
-    add_test(NAME ${TEST_NAME} COMMAND ${TEST_NAME})
+    add_test(NAME ${TEST_NAME} COMMAND ${TEST_NAME} WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}")
 endmacro()
 
 # ============================================================================
